docs/source/project_documentation/user_guide/model.md

```diff
@@ -0,0 +1,83 @@
+# Model Definitions in AaC
```

Comment:
Missed noticing this last night, but you need to add this to the User Guide index.rst file so that it is correctly exposed. 

docs/source/project_documentation/user_guide/index.rst

Reply:
Resolved in e3672be